test/reputation-update.js

```diff
@@ -196,7 +196,7 @@ contract("Reputation Updates", accounts => {
 
       const repLogEntryWorker2 = await inactiveReputationMiningCycle.getReputationUpdateLogEntry(4);
       expect(repLogEntryWorker2.user).to.equal(WORKER);
-      expect(repLogEntryWorker2.amount).to.eq.BN(WORKER_PAYOUT);
+      expect(repLogEntryWorker2.amount).to.eq.BN(WORKER_PAYOUT.muln(3).divn(2));
```

Comment:
Shouldn't the worker here get 1.5 times their payout because evaluator failed to rate them?